src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -55,11 +55,10 @@ export default function SavedObjectFactory(es, kbnIndex, Promise, Private, Notif
     let customInit = config.init || _.noop;
 
     // optional search source which this object configures
-    self.searchSource = config.searchSource && new SearchSource();
+    self.searchSource = config.searchSource ? new SearchSource() : null;
```

Comment:
> but I'm +100 to also explicitly assigning it when appropriate

Me too. When do you think it's appropriate?

I personally feel like `undefined` is only appropriate for things that are uninitialized or where the correct value can not be determined. `undefined` === "i don't know" `null` === "nothing"